persist-cli/src/main/java/io/ballerina/persist/models/ForeignKey.java

```diff
@@ -17,54 +17,16 @@
  */
 package io.ballerina.persist.models;
 
+import java.util.List;
+
 /**
  * Foreign Key class.
- *
+ * @param name name of the foreign key
+ * @param columnNames list of columns taking part in the foreign key constraint
+ * @param referenceTable reference table
+ * @param referenceColumns list of key columns in the reference table
  * @since 0.4.0
  */
-public class ForeignKey {
```

Comment:
Do we need this class?

Reply:
The information structure is used for easier extraction of required foreign key meta data from Relation when processing queries after finding the differences. The retrieval of these metadata is not straightforward so I converted this into a record. Shall I keep it as a record or use Relation?